packages/graphql/mixin.core.js

```diff
@@ -14,6 +15,25 @@ function exists(path) {
   }
 }
 
+function contextRelativeExternals(externals) {
+  return function(context, request, callback) {
+    if (externals.includes(request)) {
+      const absolutePath = require.resolve(request, {
+        paths: require.main.paths.concat(context),
+      });
+      if ((absolutePath.match(/node_modules/g) || []).length > 1) {
+        const relativePath = relative(
+          this.config.rootDir,
```

Comment:
`this.config` was `undefined` in my local test. Maybe it makes sense to pass that (or the `rootDir`) in as the first argument.